### PR TITLE
Deploy qpid client certificate and key

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Just realized that the qpid manifest generates client certs/keys.

**Comment by ehelms** [comment]
Although, the certs in qpid look they are designed for the broker itself so I am not sure why they were ever named as client cert/key for qpid.

File: manifests/qpid_client.pp [hunk 1]
```diff
@@ -8,8 +8,13 @@
...
 
   $messaging_client_cert = $::certs::messaging_client_cert,
+
+  $qpid_client_cert = $::certs::params::qpid_client_cert,
+  $qpid_client_key = $::certs::params::qpid_client_key,
 ) {
 
+  $client_cert_name = "${hostname}-qpid-client-cert"
+
   cert { "${hostname}-qpid-client-cert":
```
Comment (ekohl) [inline manifests/qpid_client.pp]: Use ```$client_cert_name here```? Doesn't that also result in a duplicate ```Cert``` resource?

File: manifests/qpid_client.pp [hunk 2]
```diff
@@ -28,18 +33,37 @@
...
+    org           => 'qpid',
+    org_unit      => $::certs::org_unit,
+    expiration    => $::certs::expiration,
+    ca            => $::certs::default_ca,
+  }
+
   if $deploy {
 
-    Cert["${hostname}-qpid-client-cert"] ~>
+    Cert["${hostname}-pulp-qpid-client-cert"] ~>
```
Comment (ekohl) [inline manifests/qpid_client.pp]: Where is this resource defined?

**Comment by ekohl** [comment]
Maybe use the specs from https://github.com/Katello/puppet-certs/pull/148 to get some testing?

**Comment by ekohl** [comment]
Needs a rebase.

**Comment by ekohl** [comment]
Ping :)

**Pull request closed without merging**